index.html

```diff
@@ -1,10 +1,35 @@
+<!DOCTYPE html>
 <html lang="en">
 <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+<meta charset="UTF-8">
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+<title>Intro Page - Top 5 National Parks</title>
+<link rel="stylesheet" href="./assets/style.css">
 </head>
+
 <body>
+  <header>
+    <img src="logo.png(example)" alt="National Park Service Logo">
+    <h1>Top 5 National Parks to Visit</h1>
+    <button id="menu-button">menu drop</button>
+  </header>
+  
+  <main>
+    <section id="intro-page">
+      <h2>INTRO PAGE</h2>
+      <p>
+        <!-- Introductory text go here. -->
+      </p>
+      <button id="start-button">START</button>
+    </section>
     
+    
+  </main>
+
+  <footer>
+    <p>National Park Service</p>
```

Comment:
I'll also add a hyperlink here connected to the national park words as well when I add the info of the parks